t/14-isotovideo.t

```diff
@@ -133,6 +133,9 @@ subtest 'isotovideo with wheels' => sub {
     $specfile->spurt("version: v0.1\nwheels: [https://github.com/foo/bar.git]");
     my $utils_mock = Test::MockModule->new('OpenQA::Isotovideo::Utils');
     my @repos;
+    my $failed_once = 0;
+    $utils_mock->redefine(clone_git => sub (@) { bmwqemu::diag "Connection reset by peer" unless $failed_once++; $failed_once });
+    combined_like { checkout_wheels("$wheels_dir") } qr@Clone failed, retries left: 0 of 2@;
```

Comment:
https://github.com/os-autoinst/os-autoinst/actions/runs/3854120671/jobs/6567769358#step:3:617 from the latest run on master shows that t/14-isotovideo.t already takes a significant 43s and now https://github.com/os-autoinst/os-autoinst/actions/runs/3876337077/jobs/6610044151#step:3:615 shows 47s. Please ensure that your changes don't increase the runtime, at best even decrease it. As you are changing code in OpenQA/Isotovideo/Utils.pm please consider extending the according unit test module https://github.com/os-autoinst/os-autoinst/blob/master/t/20-openqa-isotovideo-utils.t instead of 14-isotovideo.t